src/main/java/seedu/duke/commands/ListCommand.java

```diff
@@ -78,9 +79,8 @@ public void execute(TaskList tasks) throws DukeException {
             TaskList newTaskList = new TaskList(newTasks);
             newTaskList.listTask(category);
         } else {
-            for (int i = 0; i < tasks.size(); i++) {
-                newTasks.add(tasks.get(i));
-            }
+            newTasks = tasks.getTaskList();
+            Collections.sort(newTasks);
             TaskList newTaskList = new TaskList(newTasks);
             newTaskList.listTask();
```

Comment:
I was thinking that if we only print a new sorted list using the "list" command, the users will never know which task they are setting as "done" or "deleted".

Reply:
Ya. Maybe we can have this command as "list tasks sorted" while keeping the original "list" command as unchanged?

Reply:
Yea that makes sense.